elixir/1.11/Dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM hexpm/elixir:1.11.0-erlang-22.3.4.12-debian-buster-20200607
```

Comment:
Do we have to pin to the sub-patch Erlang version? Why not `1.11.0-erlang-23.1-debian-buster-20200607`?